cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -125,23 +125,58 @@ class Json(ListablePropertyType):
     _type = "json"
 
 
+@dataclass(frozen=True)
+class UnitReference:
+    external_id: str
+    source_unit: str | None = None
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        output = {"externalId" if camel_case else "external_id": self.external_id}
+        if self.source_unit:
+            output["sourceUnit" if camel_case else "source_unit"] = self.source_unit
+        return output
+
+    @classmethod
+    def load(cls, data: dict) -> UnitReference:
```

Comment:
Should this be?
```suggestion
    def _load(cls, data: dict) -> UnitReference:
```

Reply:
For example, `DirecRelationReference` which is a very similar class also has a public load, I don't see why it should be private